build_scripts/build_usd.py

```diff
@@ -868,7 +868,7 @@ def InstallBoost(context, force, buildArgs):
     TBB_URL = "https://github.com/oneapi-src/oneTBB/archive/2018_U6.tar.gz"
 
 # Note: this refers to a fork of tbb for wasm. Is this maintained?
-TBB_EMSCRIPTEN_URL = "https://github.com/hpcwasm/wasmtbb/archive/master.zip"
+TBB_EMSCRIPTEN_URL = "https://github.com/sdunkel/wasmtbb/archive/master.zip"
```

Comment:
The url does not point to a zip file. "https://github.com/sdunkel/wasmtbb/archive/refs/heads/master.zip" seems to work.
```suggestion
TBB_EMSCRIPTEN_URL = "https://github.com/sdunkel/wasmtbb/archive/refs/heads/master.zip"
```

Reply:
thanks - that's strange. But why does it work then? :-) It did not work for you?

Reply:
No it didn't work for me. Curl would produce a 404. I am on windows WSL, maybe that makes a difference somehow. Or the urls have changed.

Reply:
thanks, it looks like something has changed indeed